lib/msal-browser/src/interaction_client/BaseInteractionClient.ts

```diff
@@ -63,4 +64,65 @@ export abstract class BaseInteractionClient {
             }
         }
     }
+
+    /**
+     * Initializer function for all request APIs
+     * @param request
+     */
+    protected initializeBaseRequest(request: Partial<BaseAuthRequest>): BaseAuthRequest {
+        this.logger.verbose("Initializing BaseAuthRequest");
+        const authority = request.authority || this.config.auth.authority;
+
+        const scopes = [...((request && request.scopes) || [])];
+
+        // Set authenticationScheme to BEARER if not explicitly set in the request
+        if (!request.authenticationScheme) {
+            request.authenticationScheme = AuthenticationScheme.BEARER;
+            this.logger.verbose("Authentication Scheme wasn't explicitly set in request, defaulting to \"Bearer\" request");
+        } else {
+            this.logger.verbose(`Authentication Scheme set to "${request.authenticationScheme}" as configured in Auth request`);
+        }
+
+        const validatedRequest: BaseAuthRequest = {
+            ...request,
+            correlationId: this.correlationId,
+            authority,
+            scopes
+        };
+
+        return validatedRequest;
+    }
+
+    /**
+     *
+     * Use to get the redirect uri configured in MSAL or null.
+     * @param requestRedirectUri
+     * @returns Redirect URL
+     *
+     */
+    protected getRedirectUri(requestRedirectUri?: string): string {
+        this.logger.verbose("getRedirectUri called");
```

Comment:
Aside: I think we should start logging these under `trace` instead of `verbose`.